client/src/App.jsx

```diff
@@ -1,31 +1,24 @@
-import React, {Component} from 'react';
-import {BrowserRouter,Route} from 'react-router-dom';
-import { viewPopup, editPopup, deletePopup } from './Components/CommonComponent/table/Popups';
-import TableComponent from './Components/CommonComponent/table/Table';
-import SideBar from './Components/CommonComponent/Sidebar';
-import './App.css';
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+// import TableComponent from "./Components/CommonComponent/table/Table";
+// import SideBar from "./Components/CommonComponent/Sidebar";
+import Login from "./Components/Layouts/Login";
+import "./App.css";
 
-import Home from './Components/Layouts/Home';
+// import Home from "./Components/Layouts/Home";
 
 class App extends Component {
-  state = {};
+  state = {
```

Comment:
remove unnecessary code 
```suggestion
  state = {
```